### ISSUE TITLE
Drift detection: explicitly mark unchecked resources as "not checked", rather than skip from results table

### ISSUE THREAD

**Issue by plumdog** [aws-cloudformation/cloudformation-coverage-roadmap#1723]
### Name of the resource

Other

### Resource name

n/a

### Description

I think the existence of the warning
> Only resources which currently support drift detection are displayed here. To view all of your stack resources, see your stack details page.

indicates a usability bug.

Should instead include all of a stack's resources in the drift results table, and mark those that Cloudformation's drift detection doesn't support with a "not checked" warning message.

For stacks with a very small number of resources, spotting which of a stack's resources aren't in the drift results table is easy, but this quickly becomes error prone as a stack has more resources.

**Comment by FarrOut** [comment]
i tend to agree. i little bit of visibility can go a long way in preventing unnecessary confusion and false-assumptions.
